### PR TITLE
Contributing

### PR DESCRIPTION
**Pull request by eyaly**
@nadvolod - please review the image injection changes.
1. Create the 2 tests (android  and ios) in one file
2. Change the demo app

### CODE REVIEW

**Comment by nadvolod** [comment]
@c3ccl3ston would you do a review for us please?

**Review by c3ccl3ston — APPROVED** [review]
Looks good @eyaly. My only thought would be that maybe in the actual example code we should remove the Thread.sleeps as this goes against best practices and I would be concerned that users just copy/paste this code. With that said, I am going to approve since it states that those are not needed and for video purposes only.

File: appium-examples/src/test/java/com/realdevice/image_injection/ImageInjectionTest.java [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+    }
+
+
+    // Android
+    public boolean isAndroidBrowserOpened() throws InterruptedException {
+        int wait = 8;
+        int counter = 0;
+
+        do {
+            if (!androidDriver.currentActivity().contains(".MainActivity")){
```
Comment (nadvolod) [inline appium-examples/src/test/java/com/realdevice/image_injection/ImageInjectionTest.java:162]: @eyaly i'm curious why we need this if we have the `capabilities.setCapability("appWaitActivity", "com.saucelabs.mydemoapp.rn.MainActivity");` set above?

**Review by nadvolod — COMMENTED** [review]
Looks good, I just had 1 question below

File: appium-examples/src/test/java/com/realdevice/image_injection/ImageInjectionTest.java (hunk 1 above)
Comment (eyaly) [reply appium-examples/src/test/java/com/realdevice/image_injection/ImageInjectionTest.java:162]: This is about the business flow of using the image injection in the demo app. We are in the Native App, we inject the image and we should be switch to the android browser.  so this is a negative check (there is '!' in the beginning, we check that we are NOT in the native app, meaning we switch to the chrome or android browser.

**Pull request merged**